doc/changelog.md

```diff
@@ -69,6 +76,14 @@
   `__quantum__rt__device_init(int8_t *, int8_t *, int8_t *)`.
   [(#396)](https://github.com/PennyLaneAI/catalyst/pull/396)
 
+* Add `DeviceReleaseOp` to the Quantum MLIR dialect. This will be lowered to
+  the `__quantum__rt__device_release()` runtime instruction and releases the device
+  memory and update the status of the device instance from `ACTIVE` to `RELEASE`.
```

Comment:
I meant qubits :)

Reply:
Why does device release release qubits? The whole point of this instruction was to separate that concern no?

Reply:
Removed it from the support and changelog. I added the release qubits to the release-device instruction to ensure all qubits are releases before deactivating a device. This is no longer required because of our IR model that the release-device instruction always comes after releasing all qubits of the device.